k2/CMakeLists.txt

```diff
@@ -1,2 +1,3 @@
+add_subdirectory(util)
```

Comment:
please put it into csrc. No need to create a new directory.

Reply:
agreed RE no need for new directory.

Reply:
Yeap.